### PR TITLE
save gifs to camera roll

### PR DESCRIPTION
**Pull request by sergeymild**
Related to https://github.com/react-native-cameraroll/react-native-cameraroll/issues/16 issue i have made a fix to able to save Gifs to camera roll

### CODE REVIEW

**Review by HKGolden-Dev — APPROVED** [review]

**Comment by gabrieldonadel** [comment]
Is this getting merged anytime soon? @ bartolkaruza

**Review by gabrieldonadel — APPROVED** [review]

**Review by helenaford — APPROVED** [review]
Be awesome if this can get merged in, can vouch this fix works on iOS, GIFs already work on Android 💯

File: ios/RNCCameraRollManager.m [hunk 1]
```diff
@@ -137,6 +137,11 @@ static void requestPhotoLibraryAccess(RCTPromiseRejectBlock reject, PhotosAuthor
       PHAssetChangeRequest *assetRequest ;
       if ([options[@"type"] isEqualToString:@"video"]) {
         assetRequest = [PHAssetChangeRequest creationRequestForAssetFromVideoAtFileURL:inputURI];
+      } else if ([inputURI.pathExtension isEqualToString:@"gif"]) {
```
Comment (robert-bitguild) [inline ios/RNCCameraRollManager.m:140]: suggest inputURL.toLowerString, because some file like aaa.GIF

Comment (idrissakhi) [reply ios/RNCCameraRollManager.m:140]: @sergeymild would you update to lowerCase to avoid having issue, and then we can merge this PR

**Pull request closed without merging**